docs/architecture/stage_list.md

```diff
@@ -0,0 +1,1060 @@
+---
+---
+
+Stage list
+----------
```

Comment:
I think the code that generates this document should be updated to include the namespace that the stage belongs in.